cirq/ops/eigen_gate_test.py

```diff
@@ -132,22 +134,22 @@ def test_approx_eq():
         ZGateDef(exponent=1.5),
         atol=0.1
     )
-    assert not cirq.approx_eq(
-        ZGateDef(exponent=1.5),
-        ZGateDef(exponent=sympy.Symbol('a')),
-        atol=0.1
-    )
-
+    with pytest.raises(TypeError,
+                       match=re.escape("unsupported operand type(s) for"
+                                       " -: 'Symbol' and 'PeriodicValue'")):
+        cirq.approx_eq(ZGateDef(exponent=1.5),
+                       ZGateDef(exponent=sympy.Symbol('a')),
+                       atol=0.1)
     assert cirq.approx_eq(
         CExpZinGate(sympy.Symbol('a')),
         CExpZinGate(sympy.Symbol('a')),
         atol=0.1
     )
-    assert not cirq.approx_eq(
-        CExpZinGate(sympy.Symbol('a')),
-        CExpZinGate(sympy.Symbol('b')),
-        atol=0.1
-    )
+    with pytest.raises(AttributeError,
```

Comment:
it's awesome that we caught this! This was a bad test - as how can we tell if a == b? we can't!

Reply:
Ack. Thanks!